src/style/__tests__/colors.spec.js

```diff
@@ -112,38 +116,106 @@ describe('style/Colors', () => {
   });
 
   describe('generateColorPalette', () => {
+    const baseColor = '#3F88C5';
+    const tints = ['#193852', '#255379', '#316EA1', '#3F88C5', '#66A0D1', '#8DB9DD', '#B5D1E9', '#DCE9F4'];
+    const baseColorLight = '#DCE9F4';
+    const tintsLight = ['#1A3851', '#265278', '#326D9F', '#4187C3', '#68A0CF', '#8EB8DC', '#B5D1E8', '#DCE9F4'];
+    const tintsAdjustedLightness = ['#193852', '#255379', '#316EA1', '#3F88C5', '#66A0D1', '#8DB9DD', '#B5D1E9', '#DCE9F4'];
```

Comment:
It looks like there is duplication of consts, why do we need them?
`tintsAdjustedLightness` = `tints`
`tintsAdjustedSaturation` = `tintsLight`